karavan-builder/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jbangdev/jbang-action
+
+# Add Camel-JBang
+RUN jbang trust add -o --fresh --quiet https://github.com/apache/camel/blob/HEAD/dsl/camel-jbang/camel-jbang-main/dist/CamelJBang.java
+
+# Add Maven Daemon
+ADD https://github.com/mvndaemon/mvnd/releases/download/0.7.1/mvnd-0.7.1-linux-amd64.zip .
```

Comment:
There is a newer mvnd you can download from ASF
https://dist.apache.org/repos/dist/release/maven/mvnd/0.8.0/

They changed something in their release procedure so its no longer on github
https://github.com/apache/maven-mvnd/issues/623